### PR TITLE
Add interceptor that may or may not time something.

### PR DESCRIPTION
**Pull request by jschlather**
Also bumped the ELG size, because why not.

### CODE REVIEW

**Comment by acharis** [comment]
@zklapow these things cool?

File: java/grpc-client/src/main/java/io/vitess/client/grpc/TimingInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      extends ForwardingClientCall<ReqT, RespT> {
+    private final ClientCall delegate;
+    private final long start = System.currentTimeMillis();
+
+    public TimingForwardingClientCall(ClientCall delegate) {
+      this.delegate = delegate;
+    }
+
+    @Override public void sendMessage(ReqT message) {
+      long sendTime = System.currentTimeMillis() - start;
```
Comment (jhaber) [inline java/grpc-client/src/main/java/io/vitess/client/grpc/TimingInterceptor.java]: Maybe use `System.nanoTime()` instead?

File: java/grpc-client/src/main/java/io/vitess/client/grpc/netty/DefaultChannelBuilderProvider.java [hunk 2]
```diff
@@ -6,10 +6,11 @@
 import io.netty.util.concurrent.DefaultThreadFactory;
 import io.vitess.client.grpc.RetryingInterceptor;
 import io.vitess.client.grpc.RetryingInterceptorConfig;
+import io.vitess.client.grpc.TimingInterceptor;
 
 public class DefaultChannelBuilderProvider implements NettyChannelBuilderProvider {
   private static final EventLoopGroup ELG = new NioEventLoopGroup(
-      6,
+      12,
```
Comment (zklapow) [inline java/grpc-client/src/main/java/io/vitess/client/grpc/netty/DefaultChannelBuilderProvider.java]: why change the default group size?

Comment (jschlather) [reply java/grpc-client/src/main/java/io/vitess/client/grpc/netty/DefaultChannelBuilderProvider.java]: To see if it helps

**Comment by jschlather** [comment]
This is just a one-off that I'm going to revert later, so I'm just going to merge this if no one sees anything that this will break.

**Pull request merged**